_ml-commons-plugin/prebuilt-models.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title: Pre-trained models
+parent: Model-serving framework
+nav_order: 120
+---
+
+The model-serving framework supports a variety of open-source pre-built models that can assist with a variety of machine learning (ML) search and analytics use-cases. 
+
+## Uploading pre-trained models
+
+To use a pre-trained model in your OpenSearch cluster:
+
+1. Select the model and download the model artifact and config.json file. For a list of pre-trained models, see [supported pre-trained model](#supported-pre-trained-models).
+2. Upload the model using the [upload API]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-serving-framework#upload-model-to-opensearch). For more information on how to upload and use model, see [Model-serving framework]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-serving-framework).
+
+## Supported pre-trained models
+
+The model-serving framework supports the following models. 
+
+Although models might have similar use cases, they many of them contain different performance metrics and model sizes. For a comparison of the performances of pre-trained models, see the [sbert documentation](https://www.sbert.net/docs/pretrained_models.html#model-overview).
```

Comment:
Is there a way to weave this into the content? The previous line says "...supports the following models." but the next object on the page is this warning text, and it's followed by an h3 section that doesn't really tie in to the previous section. Are "Sentence transformers" the "models supported by the model-serving framework"? Or are the rows in the subsequent table the supported models? It isn't clear to me as a reader.

Reply:
I added a "categorized based on type" qualifier here. I'm breaking a rule I have where I don't like to add single nested levels in headers, but the plan is to add more pre-trained model types (such as text embedders) in each subsequent release. So, as opposed to adding a "Type" column and having a giant table in the future, and to prevent repeating the same description over and over again, we'll add a new table and a new h3 header for each model type.